pkg/processor/trigger/kafka/trigger.go

```diff
@@ -461,3 +493,28 @@ func (k *kafka) resolveSCRAMClientGeneratorFunc(mechanism sarama.SASLMechanism)
 		return nil
 	}
 }
+
+func (k *kafka) signalWorkerTermination(workerTerminationCompleteChan chan bool) {
+
+	// signal all workers on re-balance
+	k.Logger.Debug("Signaling all workers to drop or ack events due to rebalance")
```

Comment:
either reflect the "how" on function signature or let the callee determine the "how" by function arguments